scripts/tests/provisioning/test_safemode.expect

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/expect -f
+#
+# Start a safemode NILRT VM and instrument installing and running the provisioning
```

Comment:
Should this be referring to grub gateway ptests and not provisioning ptests? Are they the same? Or generically leave it at ptests?

Reply:
They should be the same right? I think keeping "provisioning ptests" which implies we're testing provisioning is more helpful than just "ptests".